package.json

```diff
@@ -147,7 +181,7 @@
         "visible": true
       },
       {
-        "permissionName": "ui-tenant-settings.settings.servicepoints",
+        "permissionName": "ui-tenant-settings.settings.servicepoints.view",
         "displayName": "Settings (tenant):  Can create, edit and remove service points",
```

Comment:
this display name should probably be renamed to "Settings (tenant):  Can view service points"!